region/norfair/west/Crocomire Speedway.json

```diff
@@ -397,6 +475,49 @@
       ],
       "devNote": "FIXME: R-Mode can be used instead of a Crystal Flash but that depends on having more Reserve Energy than Regular Energy."
     },
+    {
+      "link": [2, 4],
+      "name": "G-Mode Setup - Get Hit By Multiviola, Complex Manipulation",
+      "requires": [
+        "h_heatProof",
+        "canBePatient",
+        "canCameraManip",
+        "SpeedBooster",
+        {"or": [
+          "canTrickyDodgeEnemies",
+          {"enemyDamage": {"enemy": "Multiviola", "type": "contact", "hits": 3}}
+        ]},
+        {"or": [
+          "canTrickyUseFrozenEnemies",
```

Comment:
How do you continue breaking the speed blocks and not kill the viola without ice?

Reply:
I can add it to the note. I wasn't sure if it was worth including all 5 setups or not. I felt like it was nice to know that if you shinecharge too far then you are still fine, but maybe its unhelpful?

Reply:
I like having the possibilities documented.  Watching a clip is going to be better than reading whatever is written in the note, imo.  If the options get documented in some other way, they can be removed from the note making it more readable.
Moving the viola off camera so you can run under it is the intuitive way you'd do it so I don't think it needs to be said.